### PR TITLE
SDD-916 - Friendly WAF Error Message

### PR DESCRIPTION
**Pull request by MrKevJoy**
- Added WAF friendly error page.

### CODE REVIEW

File: src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-l">Sorry, there is a problem with the service</h1>
+        <p class="govuk-body">The information you have entered may not have been saved.  You may have to enter the information again.</p> <p>If you continue to have a problem, contact the Course Directory Support Team.</p>
+        <p class="govuk-body">
+            <strong>Course Directory Support Team</strong>
+        </p>
+        <p class="govuk-body">
+            Call 0844 811 5073 or email <a href="mailto:support@coursedirectory.org.uk"> support@coursedirectory.org.uk</a>.
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml:9]: Possibly need a CSS class here?
```suggestion
            Call 0844 811 5073 or email <a href="mailto:support@coursedirectory.org.uk" class="govuk-link"> support@coursedirectory.org.uk</a>.
```

File: src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+            <br>
+            <br>
+            Monday to Friday: 9am to 5pm.
+        </p>
+
+        <p class="govuk-body">
+            Return to  <a href="https://coursedirectory.nationalcareers.service.gov.uk/"> Publish to the course directory</a>.
```
Comment (gunndabad) [inline src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml:16]: As above

**Review by gunndabad — APPROVED** [review]

File: src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml (hunk 2 above)
Comment (MrKevJoy) [reply src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml]: Changed

File: src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml (hunk 1 above)
Comment (MrKevJoy) [reply src/Dfc.CourseDirectory.WebV2/Features/Error/WAFError.cshtml]: changed

**Pull request merged**